bundles/org.openhab.binding.mffan/README.md

```diff
@@ -0,0 +1,69 @@
+# MfFan Binding
+
+This binding is used to enable communications between openHAB and "Modern Forms" or "WAC Lighting" WIFI connected, smart, ceiling fans.  
+
+
+## Supported Things
+
+Supported things include: "Modern Forms" and "WAC Lighting" smart ceiling fans.  The binding has been tested with both.
+
+## Discovery
+
+Auto discovery is not supported at this time.  
+To bind with a smart fan either the IP address of the fan, or the local host name of the fan must be provided.
+The binding has been tested using an IPV4 local network address, and a host name served by a local DNS.
```

Comment:
```suggestion
```
Not sure if this should be mentioned at all as the configuration explains hsotname/ipadres (maybe add IPV4 to that descritpion)